common/src/main/java/tachyon/security/authentication/PlainSaslServer.java

```diff
@@ -213,7 +213,7 @@ public void handle(Callback[] callbacks) throws IOException, UnsupportedCallback
   /**
    * An instance of this class represents a client user connecting to Tachyon service.
    *
-   * It is maintained in a ThreadLocal variable based on the Thrift RPC mechanism.
+   * It is maintained in a {@link ThreadLocal} variable based on the Thrift RPC mechanism.
```

Comment:
@aaudiber The idea is a little bit different and does not have something to do with refactoring in the first place. I link them the first time they are mentioned in the documentation for a method. The reason is that if someone reads the docs and wants to jump right into that class without scrolling to the top of the file and search for the place where the class is mentioned the first time.